src/lib.rs

```diff
@@ -257,7 +263,8 @@ impl StackMapParser {
 
     /// Make an iterator over the stackmap record entries in the stackmap section.
     ///
-    /// If the iterator returns an error, the iterator becomes invalid.
+    /// If the iterator returns an error, the iterator becomes invalid and reuse will invoke
```

Comment:
I'm not sure you can "invoke" undefined behaviour, unless it's a sort of Lord O' The Rings sea shanty :) How about "lead to"?

Reply:
I'd never thought of it that way. But it does sound quite ye olde worldy, now you mention it.

FWIW, both clang and gcc call it that. Google it and you can find errors like:
```
warning: iteration 3u invokes undefined behavior [-Waggressive-loop-optimizations]
```

I don't mind your suggestion either.

Reply:
"Cause" is another alternative.